handbook/security.md

```diff
@@ -0,0 +1,256 @@
+* Firewall logging allows us to troubleshoot and investigate, by letting us know if applications or connections are being blocked by it.
+
+**User experience impact**
+
+* A password will be needed as soon as the laptop is turned on, due to the encryption process, instead of once the laptop has booted.
+* No performance impact - macOS encrypts the system drive by default. 
+* Unsigned or unnotarized applications will require extra steps to execute.
+* Unsigned applications will not be allowed to open a firewall port for inbound connections.
+
+#### Screen saver and automatic locking
+
+| #     | Setting                                                                             |
+| ----- | ----------------------------------------------------------------------------------- |
+| 2.3.1 | Ensure an inactivity interval of 20 minutes or less for the screen saver is enabled |
+| 6.1.2 | Ensure show password hints is Disabled                                              |
+| 6.1.3 | Ensure guest account is disabled                                                    |
+| NA    | Prevent the use of automatic logon                                                  |
+
+**Why?**
+
+* Our workstations are laptops, which by definition are portable and used in many different areas. If a laptop is forgotten, while logged in, it defeats the purpose of other controls and exposes data that could be critical. 
+* Password hints can be dangerous, as they can sometimes be easier to guess than the password itself. Since we can support employees remotely via MDM, and since we do not require passwords that expire, we eliminate that risk by hiding the hints.
+* Guest accounts are not useful for systems that are used by a single employee, therefore, we disable them.
+* Automatic logon would defeat the purpose of even requiring passwords to unlock computers.
+
+**User experience impact**
+
+* Laptops will lock after 20 minutes of inactivity. To voluntarily pause this, you can configure a [hot corner](https://support.apple.com/en-mo/guide/mac-help/mchlp3000/mac) to disable the screen saver. This can be useful if you are, for example, watching an online meeting without moving the mouse and want to be sure the laptop will not lock.
+* Forgotten passwords will have to be fixed via MDM, instead of relying on potentially dangerous hints.
+* Guest accounts will not be available.
+
+#### iCloud
+We do not apply ultra restrictive [Data Loss Prevention](https://en.wikipedia.org/wiki/Data_loss_prevention_software) style policies to our workstations. A computer that is used for day to day work, with full access to the Internet can never be protected from voluntary malicious actors willing to upload data. Instead, we focus on ensuring the most critical data never reaches our laptops, so it can remain secure, while our laptops can remain productive.
```

Comment:
More like, malware and people controlling the malware using your laptop to steal data. But it could also be a malicious employee - for example in banks they usually block USB devices completely because of their threat model BUT there always are so many other ways to exfiltrate data that it's usually just annoying and does not even improve security much when companies do that.

Reply:
Gotcha regarding malicious actors, but I'm wondering whether we need to include it. It's quite full-on, and hard for me to digest as an end-user/employee who just navigated here to check out our policy on iCloud usage.

I think this is easier to digest and more consistent with the other profile settings throughout the doc (i.e this is the only settings table with an intro paragraph)

> We do not apply ultra restrictive [Data Loss Prevention](https://en.wikipedia.org/wiki/Data_loss_prevention_software) style policies to our devices. Instead, we focus on ensuring the most critical data never reaches our laptops, so it can remain secure, while our laptops can remain productive.

Can we use that instead?

Reply:
Even that's a bit confusing though. "What the heck is the critical data, and why does it never reach my laptop?" thought Bob from HR.

How about:

> We do not apply ultra restrictive Data Loss Prevention style policies to our devices. Instead, by using our company Google Drive, we ensure that the most critical company data never reaches our laptops, so it can remain secure, while our laptops can remain productive.

Then eventually link Google Drive to its relevant section of the handbook when it's been added.

(Assuming that I've understood all this correctly.)